packages/strapi-design-system/src/Tabs/Tabs.tsx

```diff
@@ -15,12 +14,16 @@ const TabButton = styled.button`
   }
 `;
 
-export const Tabs = ({ children, ...props }) => {
+interface TabsProps {
```

Comment:
This needs to extend the props of `DefaultTabsRow` because you're spreading them into it.

Also you need to export the prop types.

Reply:
Just not sure why I would need to export the TabsProps interface here. Isn't it only used in this file?

Reply:
Because these types will eventually be consumed by other components e.g. in the admin panel and it's good practice to instead of writing the type again, to create a new type from the existing type (maximum inference and it's easier to see if we change a type somewhere the effect it has through out the codebase)